tests/binary.rs

```diff
@@ -0,0 +1,93 @@
+#[cfg(feature = "binaries")]
+mod binary {
+  use assert_cmd::Command;
+  use rand::distributions::Alphanumeric;
+  use rand::{thread_rng, Rng};
+  use std::env::temp_dir;
+  use std::fs::File;
+  use std::io::Read;
+  use std::path::PathBuf;
+
+  fn get_y4m_input() -> Vec<u8> {
+    let mut input = File::open(&format!(
+      "{}/tests/small_input.y4m",
+      env!("CARGO_MANIFEST_DIR")
+    ))
+    .unwrap();
+    let mut data = Vec::new();
+    input.read_to_end(&mut data).unwrap();
+    data
+  }
+
+  fn get_tempfile_path(extension: &str) -> PathBuf {
+    let mut path = temp_dir();
+    let filename =
+      thread_rng().sample_iter(&Alphanumeric).take(12).collect::<String>();
```

Comment:
Might be personal preference, but I think tests should avoid using random values. A second opinion would be nice.

Reply:
We could use a directory and use non-random filenames.